discovery-provider/src/trending_strategies/default_underground_trending_tracks_strategy.py

```diff
@@ -0,0 +1,50 @@
+from datetime import datetime
+from dateutil.parser import parse
+from src.trending_strategies.base_trending_strategy import BaseTrendingStrategy
+from src.trending_strategies.trending_type_and_version import TrendingType, TrendingVersion
+
+b = 5
+qw = 50
+hg = 1
+ie = 0.25
+pn = 0.01
+u = 30.0
+qq = 0.001
+oi = 20
+nb = 750
+
+class DefaultUndergroundTrendingTracksStrategy(BaseTrendingStrategy):
```

Comment:
It looks like the default class here has the exact same values as the ePWJD variant - we should strive to avoid duplication here, especially since there are so many tiny ways these files could accidentally diverge. I don't think we actually need a default class at all -  we could just have the named variants, and then the factory is responsible for knowing which of the named variants is currently default.

Reply:
agree, let's remove default from the strategies themselves and just makes all strategies, just plain strategies